packages/babel-plugin-transform-regexp-constructors/__tests__/fixtures/escape/actual.js

```diff
@@ -0,0 +1 @@
+String.raw`var x = new RegExp('\\w+\\s')`;
```

Comment:
remove string.raw

Reply:
It was rebase screw up.

Reply:
but why is regex constructors in your pr ?